cornellh4i/ithaca-recovery
# Atoms/box text

### Description
Implemented a BoxText component that can take on two forms: Meeting Blocks and Room Blocks.

### Testing
1. Navigate to http://localhost:3000/test
2. Scroll to the bottom of the page, and observe the added Meeting and Room Blocks
3. You can adjust the color, time, meeting name, etc. in the test page

### Notes
![Screenshot (62)](https://github.com/user-attachments/assets/0a0237d5-644b-4752-9132-7fc8e4365460)
![Screenshot (63)](https://github.com/user-attachments/assets/fe255acc-67f1-4f62-95b6-bad3aacb5115)

Related to issue #36
